### PR TITLE
Snap resizing

### PR DESCRIPTION
**Pull request by mattosborn**
listen for set-height message from snap iframes (as we do for other interactive embeds)

### CODE REVIEW

File: static/src/javascripts/projects/facia/modules/ui/snaps.js [hunk 1]
```diff
@@ -84,15 +99,17 @@ define([
...
+                height: Math.min(Math.max(spec.height || 0, minIframeHeight), maxIframeHeight)
+            }),
+            iframe = bonzo.create(iframeHtml)[0];
+
+        snapIframes.push(iframe);
+        bindIframeMsgReceiverOnce();
 
         fastdom.write(function () {
-            bonzo(el).html(source);
+            bonzo(el).empty().append(iframe);
```
Comment (phamann) [inline static/src/javascripts/projects/facia/modules/ui/snaps.js]: Why do you need `empty().append()` vs `html()`?

File: static/src/javascripts/projects/facia/modules/ui/snaps.js [hunk 2]
```diff
@@ -84,15 +99,17 @@ define([
         var spec = bonzo(el).offset(),
             minIframeHeight = Math.ceil((spec.width || 0) / 2),
             maxIframeHeight = 400,
-            source = template( // Wrapping iframe to fix iOS height-setting bug
```
Comment (phamann) [inline static/src/javascripts/projects/facia/modules/ui/snaps.js]: Is this not still relevant?

**Comment by phamann** [comment]
:+1: Other than comments.

File: static/src/javascripts/projects/facia/modules/ui/snaps.js (hunk 1 above)
Comment (mattosborn) [inline static/src/javascripts/projects/facia/modules/ui/snaps.js]: `html` takes a string whereas `append` takes an element

**Pull request merged**